src/System.CommandLine/Binding/SpecificSymbolValueSource.cs

```diff
@@ -16,23 +16,23 @@ public SpecificSymbolValueSource(IValueDescriptor valueDescriptor)
         public IValueDescriptor ValueDescriptor { get; }
 
         public bool TryGetValue(IValueDescriptor valueDescriptor,
-            BindingContext bindingContext,
-            out object boundValue)
+            BindingContext? bindingContext,
```

Comment:
Pausing my review here. Will resume on Monday on a proper computer (tablet is challenging) ;-)